packages/chain-adapters/src/ChainAdapterCLI.ts

```diff
@@ -98,18 +98,12 @@ const getWallet = async (): Promise<NativeHDWallet> => {
 // @ts-ignore:nextLine
 const testBitcoin = async (wallet: NativeHDWallet, broadcast = false) => {
   const chainAdapter = adapters.btc
-  const bip44Params: BIP44Params = {
-    purpose: 84,
-    coinType: 0,
-    accountNumber: 0,
-    isChange: false,
-    index: 10,
```

Comment:
Not sure why the previous implementation was using the 10th address index, this achieves a similar testing without it (with a different address obviously)